src/test/java/com/puppycrawl/tools/checkstyle/filters/XpathFilterElementTest.java

```diff
@@ -63,7 +64,7 @@ protected String getPackageLocation() {
     public void testMatching() throws Exception {
         final String xpath = "/CLASS_DEF[./IDENT[@text='InputXpathFilterElementSuppressByXpath']]";
         final XpathFilterElement filter = new XpathFilterElement(
-                "InputXpathFilterElementSuppressByXpath", "Test", null, null, xpath);
+                (String) null, null, null, null, xpath);
```

Comment:
updated the tests throughout this file to try and isolate the parts that are being tested, e.g. the xpath expression here. I thought it might make the tests clearer, but not all of those changes are necessary for full coverage - happy to revert/adjust